plugins/frontend/backstage-plugin-datadog/package.json

```diff
@@ -5,12 +5,29 @@
   "types": "src/index.ts",
   "license": "Apache-2.0",
   "publishConfig": {
-    "access": "public",
-    "main": "dist/index.esm.js",
-    "types": "dist/index.d.ts"
+    "access": "public"
+  },
+  "exports": {
+    ".": "./src/index.ts",
+    "./alpha": "./src/alpha.ts",
```

Comment:
Is this the naming convention for this? We need to eventually modify this to be something else than `alpha` if it ever graduates to stable, or remove it completely if it doesn't. Do we have a maintenance plan to bump this (as a breaking change) into beta and stable stages?

Reply:
Yes, this is the naming convention for the system for now while it's in alpha stage (docs [here](https://backstage.io/docs/frontend-system/building-plugins/migrating#migrating-the-plugin)).Alluded to this in the PR description, but Spotify will own the process of defining the upgrade path/maintenance once the frontend system moves out of alpha.